### PR TITLE
Added lacking mention of BINDTODEVICE for srt_config_add

### PR DESCRIPTION
**Pull request by ethouris**
Fixes #2552

### CODE REVIEW

**Comment by maxsharabayko** [comment]
The issue was posted by @nhuruiew as a discussion. Copying the text here to preserve it because the discussion will be deleted.

---

Hi

**TL;DR**  in my understanding, SRTO_BINDTODEVICE has to be mentioned as a possible option in the documentation of function srt_config_add.

In the documentation here:
https://github.com/Haivision/srt/blob/master/docs/API/API-functions.md#srt_config_add
The option SRTO_BINDTODEVICE is not in the list.

However, if you follow this link
https://github.com/Haivision/srt/issues/1120
one of the bullets leads to this pull request with the description "...mainly required for supporting SRTO_BINDTODEVICE option"
https://github.com/Haivision/srt/pull/1314

the pull-request discussion has the following code example which we have tested to work (typo and obsolete comment corrected):

```c++
SRT_SOCKGROUPCONFIG data [] = {
    srt_prepare_endpoint( ... 1 ... ),
    srt_perpare_endpoint( ... 2 ... )
};

data[0].config = srt_create_config(); 
srt_config_add(data[0].config, SRTO_BINDTODEVICE, "eth0", 4);

data[1].config = srt_create_config();
srt_config_add(data[1].config, SRTO_BINDTODEVICE, "eth1", 4);

size_t len = sizeof data;
srt_connect_group(grp, data, len);

// client code must also delete the config objects
for (auto& d: data)
    srt_delete_config(d.config);

```
